roles/kubernetes/preinstall/tasks/0020-verify-settings.yml

```diff
@@ -293,6 +293,12 @@
     msg: "kata_containers_enabled support only for containerd and crio-o. See https://github.com/kata-containers/documentation/blob/1.11.4/how-to/run-kata-with-k8s.md#install-a-cri-implementation for details"
   when: kata_containers_enabled
 
+- name: Stop if gvisor_enabled is enabled when container_manager is not containerd
+  assert:
+    that: container_manager == 'containerd'
```

Comment:
The additional lines of  roles/container-engine/meta/main.yml
```
  - role: container-engine/gvisor
    when:
      - gvisor_enabled
      - container_manager in ['docker', 'containerd']
```
seem `docker` also can be container_manager.
I guess I am missing some point.

Reply:
@oomichi see the discussion on https://github.com/kubernetes-sigs/kubespray/issues/7650, initially I wanted to support all 3 of our runtimes but it turns out only `containerd` is well supported by gVisor today. `docker` is actually not able to select the `RuntimeClass` and `cri-o` support is stil not mature enough, though I'm actually hoping that might evolve in gVisor itself.